### DISCUSSION TITLE
Dbeaver connection with singlestore

### DISCUSSION THREAD

**Discussion by ibmsatya** [dbeaver/dbeaver#20688] [General]
### Description

(conn=362606) Unknown database 'mysql'

SQL Error [1049] [42000]: (conn=362606) Unknown database 'mysql'
  (conn=362606) Unknown database 'mysql'

### DBeaver Version

Community edition Version 23.1.3.202307231901

### Operating System

windows

### Database and driver

Database name and Version: SingleStore 7.8.2
URL jdbc:singlestore://servername:3306/memsql
C:\Program Files\DBeaver\plugins\singlestore-jdbc-client-1.1.8.jar

### Steps to reproduce

1. connect to the database server using DBeaver. (Success)
2. Expand the users..
3. while loading the information above error will appear.
4. Impact: unable to script the user information.
5. Issue is happening with the new version.

**Comment by LonwoLonwo** [comment]
Hello @ibmsatya 

We have better support for the SingleStore database in our PRO products: https://dbeaver.com/edition/

Including hiding unsupported features from the navigator tree. And all our PRO features.

You can try it for free: https://dbeaver.com/trial/

**Reply by ibmsatya** [reply]
It was working with community version before. latest version release of DBeaver broke it. for this reason, we can't go for Pro editions.

**Reply by LonwoLonwo** [reply]
Broke what?
We didn't change users reading for MySQL for many years. 
It is simple "SELECT * FROM mysql.user". If SingleStore is not supporting it - what can we do?
